decidim-core/app/views/layouts/decidim/_vapid_keys.html.erb

```diff
@@ -0,0 +1,3 @@
+<% if Rails.application.secrets.vapid[:enabled] %>
```

Comment:
@quinHD when the secrets haven't been adapted this code produces an error 500, could you check the presence of the key `vapid`?